src/components/detail/image-upload/FileUploader.tsx

```
@@ -0,0 +1,140 @@
+
+export function FileUploader({
+  onFilesSelected,
+  acceptedFileTypes = ['image/*', 'application/pdf'],
+  maxFiles = 5,
+  selectedFiles,
+  setSelectedFiles
+}: FileUploaderProps) {
+  const [rejectedFiles, setRejectedFiles] = useState<File[]>([]);
+
+  const onDrop = useCallback(
+    (acceptedFiles: File[], rejected: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)